changelog/fix-wp-core-deprecation-messages

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fixed
+
+strpos and str_replace deprecation messages in PHP 8.2 and 8.2
```

Comment:
```suggestion
Fix `strpos`and `str_replace` deprecation messages in PHP 8.1 and 8.2
```

Reply:
Thanks @donnapep ! Updated it here https://github.com/Automattic/sensei/pull/7028/commits/8c4349293c7e215da4e8d578a674882f971e8901. It has dismissed the review though :( Can you kindly approve it again? Thanks!